src/main/java/step2/domian/LottoMachine.java

```diff
@@ -0,0 +1,29 @@
+package step2.domian;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoMachine {
+
+    private static List<Integer> initLottoNum = IntStream.rangeClosed(0, 45).boxed().collect(Collectors.toList());
```

Comment:
```suggestion
    private static final List<Integer> initLottoNum = IntStream.rangeClosed(0, 45).boxed().collect(Collectors.toList());
```
상수에 final 키워드 붙여주세요 :)